tests/Functional/Schema/PostgreSQLSchemaManagerTest.php

```diff
@@ -366,6 +366,25 @@ public function testDropWithAutoincrement(): void
         self::assertFalse($schemaManager->tablesExist(['test_autoincrement']));
     }
 
+    public function testDropWithSchema(): void
```

Comment:
Since the code affects all platforms, the test should cover all platforms as well.

Reply:
hi @morozov 

I'm OK with this, but I'm stucked because I don't know how to resolve [this problem](https://github.com/doctrine/dbal/pull/6576#discussion_r1823972507)

Reply:
I'm moved it in `SchemaManagerFunctionalTestCase` and I've conditionned it to `$platform->supportsSchemas()`, so that it will only run for postgreSQL / SQLServer